win32ss/user/ntuser/power.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * PROJECT:         ReactOS Win32k subsystem
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Power management of the Win32 kernel-mode subsystem
+ * COPYRIGHT:       Copyright 2024 George Bișoc <george.bisoc@reactos.org>
+ */
+
+#pragma once
+
+//
+// Power Callout type enumeration
+//
+typedef enum tagPOWER_CALLOUT_TYPE
+{
+    POWER_CALLOUT_EVENT = 0,
+    POWER_CALLOUT_STATE
+} POWER_CALLOUT_TYPE;
+
+//
+// Power Callout structure
+//
+typedef struct tagWIN32POWERCALLOUT
+{
+    /* Links the power callout with the global queue list */
+    LIST_ENTRY Link;
+
+    /* Set to POWER_CALLOUT_STATE if the upcoming power callout is a power state (IntHandlePowerState does that), POWER_CALLOUT_EVENT otherwise  */
```

Comment:
Typically you do try to respect 80 chars. I am not a zealot in this regard, but why not here? This is the longest line in this file.